example/src/release_tests/mouseButtons/index.tsx

```diff
@@ -0,0 +1,206 @@
+import React from 'react';
+import {
+  Gesture,
+  GestureDetector,
+  GestureType,
+  MouseButton,
+  Directions,
+  ScrollView,
+} from 'react-native-gesture-handler';
+import { StyleSheet, View, Text } from 'react-native';
+
+type TestProps = {
+  name: string;
+  gestureHandlers: GestureType[];
+};
+
+function Test({ name, gestureHandlers }: TestProps) {
+  return (
+    <View style={[{ height: 300 }, styles.center]}>
+      <Text> {name} </Text>
+      <View style={{ flexDirection: 'row' }}>
+        <GestureDetector gesture={gestureHandlers[0]}>
+          <View style={styles.boxLeft} />
+        </GestureDetector>
+        <GestureDetector gesture={gestureHandlers[1]}>
+          <View style={styles.boxMiddle} />
+        </GestureDetector>
+        <GestureDetector gesture={gestureHandlers[2]}>
+          <View style={styles.boxRight} />
+        </GestureDetector>
+      </View>
+      <View style={{ flexDirection: 'row' }}>
+        <GestureDetector gesture={gestureHandlers[3]}>
+          <View style={styles.boxLeftRight} />
+        </GestureDetector>
+        <GestureDetector gesture={gestureHandlers[4]}>
+          <View style={styles.boxAll} />
+        </GestureDetector>
+      </View>
+    </View>
+  );
+}
+
+function TapTests() {
+  const leftTap = Gesture.Tap()
+    .mouseButton(MouseButton.LEFT)
+    .onEnd(() => console.log('Tap with left'));
+
+  const rightTap = Gesture.Tap()
+    .mouseButton(MouseButton.RIGHT)
+    .onEnd(() => console.log('Tap with right'));
+
+  const middleTap = Gesture.Tap()
+    .mouseButton(MouseButton.MIDDLE)
+    .onEnd(() => console.log('Tap with middle'));
+
+  const leftRightTap = Gesture.Tap()
+    .mouseButton(MouseButton.LEFT | MouseButton.RIGHT)
+    .onEnd(() => console.log('Tap with left | right'));
+
+  const allTap = Gesture.Tap()
```

Comment:
These are not all buttons 😈.

Anyway, this made me think - is adding something like `MouseButton.ALL` useful?

Reply:
Given that we've assumed the left button as the default one, it sounds like it will come in handy.